workflow/automation/execution_scripts/auto_submit.py

```diff
@@ -196,7 +195,7 @@ def submit_script_to_scheduler(script_name, target_machine=None, **kwargs):
             model_params_path=params.MODEL_PARAMS,
             mgmt_db_loc=root_folder,
             run_name=run_name,
-            script_location=os.path.expandvars("$gmsim/workflow/scripts/im_plot.sl"),
+            script_location=os.path.expandvars("$gmsim/workflow/workflow/automation/org/nesi/im_plot.sl"),
```

Comment:
Erg, this needs a TODO added to allow for loading a kisti version

Reply:
Yes, I noticed this too. Was more concerned of getting this done quickly (to minimize divergence)